cleanup/mavenSnapshotCleanupWhenRelease/MavenSnapshotCleanupWhenReleaseTest.groovy

```diff
@@ -0,0 +1,65 @@
+import static org.jfrog.artifactory.client.ArtifactoryClient.create
+
+import org.jfrog.artifactory.client.Artifactory
+
+import groovyx.net.http.HttpResponseException
+import spock.lang.Specification
+
+class MavenSnapshotCleanupWhenReleaseTest extends Specification {
+
+    def TEST_FOLDER = 'org/jfrog/test'
+    def TEST_NAME = 'test-snapshot-cleanup-when-release'
+
+    def 'Maven Snapshot Cleanup When Release test'() {
+        setup:
+        def baseurl = 'http://localhost:8088/artifactory'
+        def artifactory = create(baseurl, 'admin', 'password')
+
+        deleteNoErrorIfNotFound(artifactory, 'maven-local-lib-releases', TEST_FOLDER)
```

Comment:
@axel3rd you can add an execution endpoint to the plugin to force configuration reload. You can call this endpoint after creating the repositories. Your test case can assume that the properties file (as it is in git hub) is available at the plugins folder, because our test framework will take care of that for you.